lib/win32/taskscheduler.rb

```diff
@@ -193,7 +193,30 @@ def enum
     # Returns whether or not the specified task exists.
     #
     def exists?(task)
-      enum.include?(task)
+      path = nil
+      task_name = nil
+
+      if task.include?("\\")
+        *path, task_name = task.split("\\")
```

Comment:
Add a comment here like "# Use the splat operator put all folder elements into path and leave only the task name`